qiskit/ignis/verification/entanglement/analysis.py

```diff
@@ -0,0 +1,160 @@
+
+
+"""
+This module provides several miscellaneous tools for
+analysis of the GHZ State (most notably, Fourier Analysis)
+"""
+
+
+import numpy as np
+from qiskit import *
+import matplotlib.pyplot as plt
+
+
+def ordered_list_generator(counts_dictionary, qn):
+    '''
+    For parity oscillations; just arranges dictionary of counts
+    in bitwise binary order to compute dot products more easily
+    '''
+    orderedlist = []
+    limit = 2**qn
+    keycode = str(qn + 2)
+    for a in range(limit):
+        keyq = str(format(a, '#0' + keycode + 'b'))[2:]
+        if keyq in counts_dictionary.keys():
```

Comment:
Can be replaced by one-liner:
`orderedlist.append(counts_dictionary.get(keyq, 0.0))`

Reply:
done